lib/utils/timetable.ts

```diff
@@ -59,14 +63,23 @@ export const resolveTimetableEventsCSV = (
     );
     if (!formEvent) {
       // This shouldn't ever happen
-      throw Error("Form event not found");
+      throw Error(`event ${loadedEvent.developmentPlanEvent} not found`);
+    }
+    const currentDate = getFormattedDate();
+
+    const isStatusChangeEvent = statusChangeEvents.some(
+      (event) => event === formEvent.developmentPlanEvent
+    );
+
+    if (isStatusChangeEvent) {
+      loadedEvent.endDate = currentDate;
```

Comment:
We could `return` here, which would skip to the next `forEach` loop, then we wouldn't need the `!isStatusChangeEvent` in the next `if`

Reply:
good shout! Thanks Ronan :)

Reply:
Done!